lib/rubocop/cop/packaging/relative_require_to_lib.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop # :nodoc:
+  module Cop # :nodoc:
+    module Packaging # :nodoc:
+      # TODO
+      # Write about this cop.
+      # Write bad and good methods.
+      # Help from cop/lint/redundant_require_statement.rb.
+      #
+      class RelativeRequireToLib < Cop
+        # This is the message that will be displayed when RuboCop finds an
+        # offense of using `require` with relative path to lib.
+        MSG = 'Avoid using `require` with relative path to lib.'
+
+        def_node_matcher :require?, <<~PATTERN
+          (send nil? :require
+            (str {#starts_with_relative_path?}))
+        PATTERN
```

Comment:
With this `PATTERN`, we're checking the calls in the entire source.
Whereas, what we really need to do is find these calls in `spec`, `specs`, `test`, `tests`, directory.

Perhaps using `xstr` node **might** help (something like this:)
```
(begin
  (xstr
    (str "cd spec"))
  (send nil :require
    (str #starts_with_relative_path)))
```

but I am not sure if this will work though :P

Reply:
Interesting suggestion by @deivid-rodriguez to just use the following lines in `.rubocop.yml`:
```
  Include:
    - {spec, specs, test, tests}/**/*.rb
```

So this problem is done! :fire:

Reply:
Fixed in the latest push!
Thanks, @utkarsh2102, for the quick fix! :100: